js/repos.js

```diff
@@ -447,13 +465,18 @@
                     console.log("  |-> Add team " + team.name);
                     teamAll.append($("<div>").addClass("col-xs-12").append($("<h3>").text(team.name)));
                     // Add Team in Group sub-panel
-                    teamList.append($("<li>").attr("role", "presentation")
+                    teamItem = $("<li>").attr("role", "presentation");
+                    if (team.slug == initialTeam)
+                      teamItem.addClass("active");
```

Comment:
Even one line conditional code should be surrounded with braces.

Reply:
Should I fix the other conditions above too?
https://github.com/Orange-OpenSource/Orange-OpenSource.github.io/blob/efad25ceccafa3a21c5b9c4dae5715c308a825a0/js/repos.js#L403-L406

Reply:
Yes please. Once done, I will merge this PR

Reply:
And if possible, use your Orange account instead of your public one to propose PR.